packages/react-native-reanimated/Common/cpp/reanimated/LayoutAnimations/LayoutAnimationsProxy.cpp

```diff
@@ -429,8 +433,9 @@ void LayoutAnimationsProxy::addOngoingAnimations(
     }
 
     auto &layoutAnimation = layoutAnimationIt->second;
-    layoutAnimation.opacity.reset();
-
+    if (!layoutAnimation.isViewAlreadyMounted) {
+      layoutAnimation.isViewAlreadyMounted = true;
+    }
```

Comment:
No need to perform a conditional here (most likely the compiler would automatically optimize it away anyway) 
```suggestion
    layoutAnimation.isViewAlreadyMounted = true;
```